docs/running-jobs/job-and-queue-scheduling.md

```diff
@@ -1,55 +1,128 @@
+	 - [qsub - submit a job to run](#qsub)
+	 - [qstat - query the status of jobs/queues](#qstat)
+	 - [qalter - alter a queued job](#qalter)
+	 - [qdel - delete a queued or running job](#qdel)
+	 - [qmove - move a job to a different queue](#qmove)
+	 - [qhold,qrls - place/release a hold on a job in a queue](#qhold,qrls)
+	 - [qselect - utility to select jobids that meet criteria](#qselect)
+	 - [qmsg - write a message into a jobs output file](#qmsg)
+	 - [qsig - send a signal to a job](#qsig)
+	 - [pbsnodes - Get information about the current state of nodes](#pbsnodes)
+ - **Obtaining and managing compute resources at ALCF - Machine Specific Information**
+     - [Polaris](#Polaris)
+         - [Queues](#Polaris-Queues)
+         - [Network: Rack and Dragonfly Group Mappings](#Rack-and-Dragonfly-Group-Mappings)
+         - [Compute Node Access to the Internet](#Compute-Node-Access-to-the-Internet)
+ - **Controlling task execution on your allocated resources**
+    - [Running MPI+OpenMP Applications](#Running-MPI+OpenMP-Applications)
+    - [Running GPU-enabled Applications](#Running-GPU-enabled-Applications)
+    - [Running Multiple MPI Applications on a node](#Running-Multiple-MPI-Applications-on-a-node)
+    - [Binding MPI ranks to GPUs](#Binding-MPI-ranks-to-GPUs)
+    - [Running Multiple Multi-node Applications](#Running-Multiple-Multi-node-Applications)
+    - [Using Fakeroot with Singularity](#Using-Fakeroot-with-Singularity)
+
+
+
 ## Obtaining and managing compute resources at ALCF
 
-### Definitions and notes
+### <a name="Definitions-and-Notes"></a>Definitions and Notes
 `chunk`: *A set of resources allocated as a unit to a job. Specified inside a selection directive. All parts of a chunk come from the same host.  In a typical MPI (Message-Passing Interface) job, there is one chunk per MPI process.*
 
 `vnode`: *A virtual node, or vnode, is an abstract object representing a host or a set of resources which form a usable part of an execution host. This could be an entire host,
 or a nodeboard or a blade. A single host can be made up of multiple vnodes. Each vnode can be managed and scheduled independently. Each vnode in a complex must have a unique name. Vnodes on a host can share resources, such as node-locked licenses.*  PBS operates on vnodes.  A vnode can, and in ALCF often will, represent an entire host, but it doesn't have to.  For instance, there is a mode on Polaris where we could have each physical host look like four vnodes, each with 16 threads, 1/4 of the RAM and one A100.
 
-`ncpus`: In ALCF, given the way we configure PBS, this equates to a hardware thread.  For example, Polaris has a single socket with a 32 core CPU, each with two threads, so PBS reports that as ncpus=64.
+`ncpus`: Number of resources available to execute a program. In ALCF, given the way we configure PBS, this equates to a hardware thread.  For example, a single socket node with a 32 core CPU, each with two hardware threads would report that as ncpus=64.
 
```

Comment:
Did we actually want to define "job" and "task" here, since everyone uses those differently, possibly with a mapping (like task == PALS "job"?)

Reply:
Addressed in c9d2a7cc11f2efcece42ffbf7ed4576d15e635c5